services/121-service/test/users/user-roles.test.ts

```diff
@@ -91,5 +97,101 @@ describe('/ Users', () => {
       expect(response.status).toBe(HttpStatus.BAD_REQUEST);
       expect(response.text).toMatchSnapshot();
     });
+
+    it('should return all user roles', async () => {
+      // Arrange
+
+      // Act
+      const response = await getUserRoles(accessToken);
+      const rolesLength = response.body.length;
+      console.log('rolesLength: ', rolesLength);
+
+      // Assert
+      expect(response.status).toBe(HttpStatus.OK);
+      expect(rolesLength).toBe(10);
+    });
+
+    it('should update a role by userRoleId', async () => {
+      // Arrange
+      const updateData = {
+        label: 'Updated user role label',
+        description: 'Updated user role description',
+      };
+      // Act
+      const updateUserRole = await getServer()
+        .put('/roles/1')
+        .set('Cookie', [accessToken])
+        .send(updateData);
+
+      // Assert
+      expect(updateUserRole.status).toBe(HttpStatus.OK);
+      const getUserRole = await getUserRoles(accessToken);
+
+      expect(getUserRole.status).toBe(HttpStatus.OK);
+      const role = getUserRole.body.find((r: { id: number }) => r.id === 1);
+
+      expect(role.label).toBe(updateData.label);
+      expect(role.description).toBe(updateData.description);
+    });
+
+    it('should delete a role by userRoleId', async () => {
+      // Arrange
+      // Get user roles before delete
+      const getUserRoleBeforeDelete = await getUserRoles(accessToken);
+      expect(getUserRoleBeforeDelete.status).toBe(HttpStatus.OK);
+      const rolesLengthBeforeDelete = getUserRoleBeforeDelete.body.length;
+      // Act
+      // Delete user role
+      const deleteUserRole = await getServer()
+        .delete('/roles/2')
+        .set('Cookie', [accessToken])
+        .send();
+      expect(deleteUserRole.status).toBe(HttpStatus.OK);
+      // Assert
+      // Get user roles after delete
+      const getUserRoleAfterDelete = await getUserRoles(accessToken);
+      expect(getUserRoleAfterDelete.status).toBe(HttpStatus.OK);
+      const rolesLengthAfterDelete = getUserRoleAfterDelete.body.length;
+      expect(rolesLengthAfterDelete).toBe(rolesLengthBeforeDelete - 1);
+    });
+
+    it('should get all users', async () => {
+      // Arrange
+
+      // Act
+      const getAllUsersResponse = await getAllUsers(accessToken);
+      const usersLength = getAllUsersResponse.body.length;
+
+      // Assert
+      expect(getAllUsersResponse.status).toBe(HttpStatus.OK);
+      expect(usersLength).toBe(10);
+    });
+
+    it('should get current user', async () => {
+      // Arrange
+      // Act
+      const getCurrentUserResponse = await getServer()
+        .get('/users/current')
+        .set('Cookie', [accessToken])
+        .send();
```

Comment:
is your guideline for putting this in a helper or not, whether you use it once or at least twice?